core/clients/metadataclients/metadata-clients.go

```diff
@@ -25,10 +25,14 @@ import (
 
 	"github.com/edgexfoundry/edgex-go/core/clients/types"
 	"github.com/edgexfoundry/edgex-go/core/domain/models"
+	"github.com/edgexfoundry/edgex-go/support/logging-client"
 )
 
 var (
 	ErrResponseNil error = errors.New("Problem connecting to metadata - reponse was nil")
+	ErrNotFound    error = errors.New("Item not found")
+	MetaData             = "metadata"
+	loggingClient        = logger.NewClient(MetaData, false, "")
```

Comment:
This is just plain not right. The metadata client can be used from more than one service and here we're using "Metadata" as the key. We lose visibility now to which service the client is logging on behalf of. Not only that, but the "isRemoteLogging" property is hard coded to false. I guess we'll never be able to log to a remote service with this implementation.

I should have put my foot down and refused this PR as the whole thing needs to come out and/or be redone.

Reply:
@tsconn23 this PR's purpose is replacing `fmt.Println` to `logger`. No matter use what key and no matter if `isRemoteLogging` was set to `false` or not, it would be better than using `Println`. If using `Println` you would get **nothing**.  I did not say after we finished the replacing task, there is nothing to do. There are some parts should be adjusted. This mode is ***step by step*** or ***issue by issue*** follow the plan.

Before starting this issue, I said it just remove `fmt.Println` and just the first step. Why you can tolerate a project output to console? It's not a demo.  It's first step, it always need to be done. But there are something need to be fixed as you said.

I should stop watch core and service. If anything I can help, let me know.

cc @jpwhitemn and @drasko

Reply:
@tsconn23 this PR's purpose is replacing `fmt.Println` to `logger`. No matter use what key and no matter if `isRemoteLogging` was set to `false` or not, it would be better than using `Println`. If using `Println` you would get **nothing**.  I did not say after we finished the replacing task, there is nothing to do. There are some parts should be adjusted. This mode is ***step by step*** or ***issue by issue*** follow the plan.

Before starting this issue, I said it just remove `fmt.Println` and just the first step. Why you can tolerate a project output to console? It's not a demo.  It's first step, it always need to be done. But there are something need to be fixed as you said.

I should stop watch core and service. If anything I can help, let me know.

cc @jpwhitemn and @drasko